### PR TITLE
BlueBEAR Portal Meta Modules

### PR DESCRIPTION
**Pull request by branfosj**
For the portal - meta modules. All being well, this should be very quick as all the underlying modules should already have been built.

* [x] Assigned to reviewer

`BEAR-Python-DataScience-2019b-foss-2019b-Python-2.7.16.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell

`BEAR-Python-DataScience-2019b-foss-2019b-Python-3.7.4.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell

`BEAR-Python-DataScience-2019b-fosscuda-2019b-Python-3.7.4-ppc64le.eb`
* [ ] EL7-power9

`BEAR-Python-DataScience-2019b-fosscuda-2019b-Python-3.7.4.eb`
* [ ] EL7-haswell

`BEAR-Python-Sciences-2019b-foss-2019b-Python-3.7.4.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell

`BEAR-R-bio-2019b-foss-2019b-R-3.6.2.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell

`BEAR-R-bio-2019b-fosscuda-2019b-R-3.6.2-ppc64le.eb`
* [ ] EL7-power9

`BEAR-R-bio-2019b-fosscuda-2019b-R-3.6.2.eb`
* [ ] EL7-haswell

`BEAR-R-geo-2019b-foss-2019b-R-3.6.2.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell

`BEAR-R-geo-2019b-fosscuda-2019b-R-3.6.2.eb`
* [ ] EL7-haswell

### CODE REVIEW

File: easybuild/easyconfigs/b/BEAR-Python-DataScience/BEAR-Python-DataScience-2019b-foss-2019b-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Bundle'
+
+name = 'BEAR-Python-DataScience'
+version = '2019b'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = ""
+description = """This module provides access to several Python Data Science libraries."""
```
Comment (edmondac) [inline easybuild/easyconfigs/b/BEAR-Python-DataScience/BEAR-Python-DataScience-2019b-foss-2019b-Python-2.7.16.eb:9]: We should include an encouragement to use the Python 3 equivalent.

File: easybuild/easyconfigs/b/BEAR-Python-DataScience/BEAR-Python-DataScience-2019b-fosscuda-2019b-Python-3.7.4-ppc64le.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Bundle'
+
+name = 'BEAR-Python-DataScience'
+version = '2019b'
+_pyversuf = '-Python-%(pyver)s'
+versionsuffix = '%s-ppc64le' % _pyversuf
+
```
Comment (edmondac) [inline easybuild/easyconfigs/b/BEAR-Python-DataScience/BEAR-Python-DataScience-2019b-fosscuda-2019b-Python-3.7.4-ppc64le.eb:8]: Can you include a comment explaining the strategy of having a ppc64le-specific module?

File: easybuild/easyconfigs/b/BEAR-R-bio/BEAR-R-bio-2019b-fosscuda-2019b-R-3.6.2-ppc64le.eb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Bundle'
+
+name = 'BEAR-R-bio'
+version = '2019b'
+_rversuf = '-R-%(rver)s'
+versionsuffix = '%s-ppc64le' % _rversuf
+
```
Comment (edmondac) [inline easybuild/easyconfigs/b/BEAR-R-bio/BEAR-R-bio-2019b-fosscuda-2019b-R-3.6.2-ppc64le.eb:8]: Can you include a comment explaining the strategy of having a ppc64le-specific module?

**Comment by branfosj** [comment]
> I also think it would be worth including something like "see the extension list below" at the end of each description.

I intend to add a paragraph to the application page that points people at which version they'll likely be using based on options they've selected. I'll also cover the the Python 2 encouragement in that.

**Pull request merged**